README.md

```diff
@@ -1,2 +1,10 @@
 # repetitivo
+
 A system.
+
+## Documentation
+
+Build documentation to `docs` with `npm run generate:docs`.
```

Comment:
Maybe add a slash to clarify that it's a directory? I.e. `docs/`

Reply:
Good suggestion.